tools/wpt/expectation.json

```diff
@@ -2668,8 +2668,14 @@
           "Check that no-cors request constructor is filtering headers provided as init parameter",
           "Check that no-cors request constructor is filtering headers provided as part of request parameter"
         ],
-        "request-init-contenttype.any.html": true,
-        "request-init-contenttype.any.worker.html": true,
+        "request-init-contenttype.any.html": [
+          "Default Content-Type for Request with Blob body (set type)",
```

Comment:
This fails because it expects `a/b; c=d`, but we're now setting to `a/b;c=d` as per the MIME Sniffing spec